samcli/lib/remote_invoke/exceptions.py

```diff
@@ -0,0 +1,12 @@
+"""
+Exceptions used by remote invoke executors
+"""
+from samcli.commands.exceptions import UserException
+
+
+class InvalidResourceBotoParameterException(UserException):
```

Comment:
Apologies for confusion, but any exception that we are going to use in `lib` module should not be extended from `click` library (`UserException` does extend from `click.Exception`). For that reason, we can revert this change and handle this exception seperately once we implement the command or its context.

Reply:
Oh no that makes sense, I forgot to change it back after our offline discussion yesterday. Updated in the new commit.